sdk/src/metrics/meter.cc

```diff
@@ -206,7 +206,7 @@ std::unique_ptr<WritableMetricStorage> Meter::RegisterMetricStorage(
         "[Meter::RegisterMetricStorage] - Error during finding matching views."
         << "Some of the matching view configurations mayn't be used for metric collection");
   }
-  return std::move(storages);
+  return storages;
```

Comment:
What is the compiling warning?

Reply:
The move here forbids copy elision.

Reply:
> What is the compiling warning?

`return std::move(...)` will effect NRVO.